### ISSUE TITLE
[BUG] tsSetup: inconsistent date12 order in *Stack.vrt file

### ISSUE THREAD

**Issue by yunjunz** [aria-tools/ARIA-tools#241]
**Describe the bug**

@totaibi01 and I found that the order of date12 info among `stack/*Stack.vrt` file are not consistent (409 and 412). However, this is assumed and used in MintPy [prep_aria.py](https://github.com/insarlab/MintPy/blob/main/mintpy/prep_aria.py#L477). Is this expected? 

Could you please fill the rest parts (steps to reproduce and Desktop info) @totaibi01?

**To Reproduce**

1. Download 

2. Command used

3. Error Output

stack/unwrapStack.vrt
```
    <VRTRasterBand dataType="Float32" band="409">
        <SimpleSource>
            <SourceFilename relativeToVRT="1">../unwrappedPhase/20200302_20190131.vrt</SourceFilename>
            <SourceBand>1</SourceBand>
            <SourceProperties RasterXSize="601" RasterYSize="601" DataType="Float32"/>
            <SrcRect xOff="0" yOff="0" xSize="601" ySize="601"/>
            <DstRect xOff="0" yOff="0" xSize="601" ySize="601"/>
        </SimpleSource>
```

stack/cohStack.vrt
```
    <VRTRasterBand dataType="Float32" band="412">
        <SimpleSource>
            <SourceFilename relativeToVRT="1">../coherence/20200302_20190131.vrt</SourceFilename>
            <SourceBand>1</SourceBand>
            <SourceProperties RasterXSize="601" RasterYSize="601" DataType="Float32"/>
            <SrcRect xOff="0" yOff="0" xSize="601" ySize="601"/>
            <DstRect xOff="0" yOff="0" xSize="601" ySize="601"/>
        </SimpleSource>
```

stack/connCompStack.vrt
```
    <VRTRasterBand dataType="Int16" band="412">
        <SimpleSource>
            <SourceFilename relativeToVRT="1">../connectedComponents/20200302_20190131.vrt</SourceFilename>
            <SourceBand>1</SourceBand>
            <SourceProperties RasterXSize="601" RasterYSize="601" DataType="Int16"/>
            <SrcRect xOff="0" yOff="0" xSize="601" ySize="601"/>
            <DstRect xOff="0" yOff="0" xSize="601" ySize="601"/>
        </SimpleSource>
```

**Expected behavior**

The date12 info and band index should be the same among all `stack/*Stack.vrt` files.

**Desktop (please complete the following information):**
 - ARIA-tools git tag: [e.g. git show --summary]
 - OS

**Comment by dbekaert** [comment]
@yunjunz @totaibi01, thanks for spotting this, not expected behavior indeed. Could you perhaps also upload the stack vrt files for us to have a look at?

**Comment by yunjunz** [comment]
Sure, here they are:
[cohStack.vrt.txt](https://github.com/aria-tools/ARIA-tools/files/5492172/cohStack.vrt.txt)
[connCompStack.vrt.txt](https://github.com/aria-tools/ARIA-tools/files/5492173/connCompStack.vrt.txt)
[unwrapStack.vrt.txt](https://github.com/aria-tools/ARIA-tools/files/5492174/unwrapStack.vrt.txt)

**Comment by dbekaert** [comment]
@sssangha from the initial test you believe this would have impacted the coherence base diversion in Mintpy, but should be ok when have applied closure approach? @yunjunz thoughts?

**Comment by dbekaert** [comment]
fixed per #242

**Comment by yunjunz** [comment]
Both unwrapping error correction (bridging and phase closure) uses connected components as the basic unit, so its previous potential (not in Thamer's dataset at least) un-synchronized date12 order could lead to an improper correction.

The coherence-based weighting and unwrapping error correction should be the only scenarios that could be impacted.
